packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -57,10 +68,13 @@ BuildApp() {
 
   # Default value of assets_path is flutter_assets
   local assets_path="flutter_assets"
-  # The value of assets_path can set by add FLTAssetsPath to AppFrameworkInfo.plist
-  FLTAssetsPath=$(/usr/libexec/PlistBuddy -c "Print :FLTAssetsPath" "${derived_dir}/AppFrameworkInfo.plist" 2>/dev/null)
-  if [[ -n "$FLTAssetsPath" ]]; then
-    assets_path="${FLTAssetsPath}"
+  # The value of assets_path can set by add FLTAssetsPath to
+  # AppFrameworkInfo.plist.
+  if PlistHasField "FLTAssetsPath" "${derived_dir}/AppFrameworkInfo.plist"; then
```

Comment:
Does this need to be implemented in bash? Could it be something in the tool build instead?